alerts/cluster-monitoring-operator/ThanosRuleRuleEvaluationLatencyHigh.md

```diff
@@ -26,16 +26,16 @@ oc -n openshift-monitoring logs -l 'app.kubernetes.io/name=thanos-query' \
 -c thanos-query
 ```
 
-This alert will trigger when the rule evaluation take more time than
-the scheduled interval. It can indicate that your query backend
+This alert will trigger when the rule evaluation takes more time than
```

Comment:
How about: 

> This alert is fired when one or more rule evaluation takes more than the
> maximum allowed time. An non-exhaustive list of the root cause are 
>   * Query backend (Thanos Querier) taking too long to evaluate a rule
>   * A slow Store API  
>   * Too complex rule

Why are we calling it `configured interval` ? Is it an `interval` or a  `timeout` ? 
What we do mean by a `slow StoreAPI` ? How should a user interpret this?

Reply:
configured  interval  means `interval` field in thanos rule group https://thanos.io/tip/components/rule.md/#configuring-rules

may be we can change the sentence like configured interval for rule group which will make it more clear